src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -349,6 +349,47 @@ export default function ({
     });
   }
 
+  function getConditionalTemplate(term: { value?: string; context?: AutoCompleteContext }) {
+    const context = term.context;
+    if (context?.endpoint && term.value) {
+      const { data_autocomplete_rules: autocompleteRules } = context.endpoint;
+      if (autocompleteRules && autocompleteRules[term.value]) {
+        const currentLineNumber = editor.getCurrentPosition().lineNumber;
+        const rules = (autocompleteRules[term.value] as any).__one_of ?? [];
+        const startLine = getStartLineNumber(currentLineNumber, context.requestStartRow!, rules);
+        const lines = editor.getLines(startLine, currentLineNumber).join('\n');
+        const match = matchCondition(rules, lines);
+        if (match && match.__template) {
+          return match.__template;
+        }
+      }
+    }
+  }
+
+  function getStartLineNumber(currentLine: number, endLine = 1, rules = []): number {
+    if (currentLine === 1) return currentLine;
```

Comment:
nit: 
please, change 
```js
if (currentLine === 1) return currentLine;
```

to: 
```ts
if (currentLine === 1) {
   return currentLine;
}
```

I was writing in such a way too because prettier is allowing it, but Clint Hall, Stratoula, Corey, and Alex Antonov were asking me not to write like that. Could you, please, always use the pattern with figure brackets? Thanks.

cc @alexwizp.